torch/csrc/inductor/aoti_runtime/model_container.h

```diff
@@ -409,7 +409,9 @@ class AOTInductorModelContainer {
   RAIIDataPtr constant_blob_;
   RAIIDataPtr constant_blob_secondary_;
 
+#if defined(USE_CUDA) || defined(USE_XPU)
   size_t blob_size_;
```

Comment:
This works, we could had also done `maybe-unused though

Reply:
Either way, I followed what's done in other parts of the file (looks like every use of `blob_size_` is guarded by these two defines)